wagtailmedia/tests/test_widgets.py

```diff
@@ -0,0 +1,61 @@
+import importlib
+
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.urls import reverse
+
+from wagtailmedia import widgets
+
+
+class WidgetTests(TestCase):
+    @patch.dict("sys.modules", {"wagtail.core.telepath": None})
+    def test_import_telepath_older_wagtail_versions_workaround(self):
+        importlib.reload(widgets)
+        from wagtailmedia.widgets import WidgetAdapter
+        self.assertTrue(getattr(WidgetAdapter, "fake", False))
```

Comment:
suggestion: Instead of the fake attribute, check that the core `WidgetAdapter` attribute `js_constructor` or even the `media` method doesn't exist

Reply:
You're right, that's better.